src/main/js/dashboard-frontend/src/components/details/ConfigEditor.tsx

```diff
@@ -162,19 +145,13 @@ export class ConfigEditor extends React.Component<
       call: APICall.getComponent,
       args: [this.props.service],
     })
-      .then((component) => {
-        this.setState({ service: component });
-        return component;
-      })
       .then((component) =>
         SERVER.sendRequest({ call: APICall.getConfig, args: [component.name] })
       )
       .then((config) => {
         if (config.successful) {
           this.setState({ value: config.yaml });
-          if (this.state.service.origin === USER_CREATED) {
-            this.setState({ allowEdit: true });
-          }
+          this.setState({ allowEdit: true });
```

Comment:
what's the context behind this change?

Reply:
We didn't allow you to edit the configuration for components that weren't considered user created, which was always silly. It is especially annoying when using plugins in the trusted plugin directory because those weren't considered to be user created, so you couldn't change the config. 

Without this change you could view and control any service, you just couldn't edit the config using the console. But in order to be truly useful and powerful, we should let you change the configuration too.

Reply:
I agree with the change, but could you make a separate PR for it since it's not related to the netty event loop problem?

Reply:
#22